apps/python-calibration-generator/board.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
```

Comment:
Please use short license header: https://github.com/opencv/opencv/wiki/Coding_Style_Guide#file-structure

```
// This file is part of OpenCV project.
// It is subject to the license terms in the LICENSE file found in the top-level directory
// of this distribution and at http://opencv.org/license.html.
```

Use `#` instead of `//` in Python code.

Reply:
changed.